src/hdf5_interface.cpp

```diff
@@ -734,34 +745,34 @@ write_string(hid_t group_id, int ndim, const hsize_t* dims, size_t slen,
 
 
 void
-write_string(hid_t group_id, const char* name, const std::string& buffer, bool indep)
+write_string(hid_t group_id, const char* name, const std::string& buffer,
+             bool indep)
 {
-  write_string(group_id, 0, nullptr, buffer.length(), name, buffer.c_str(), indep);
+  write_string(group_id, 0, nullptr, buffer.length(), name, buffer.c_str(),
+               indep);
 }
 
 
 void
-write_tally_results(hid_t group_id, hsize_t n_filter, hsize_t n_score, const double* results)
+write_tally_results(hid_t group_id, hsize_t n_filter, hsize_t n_score,
+                    const double* results)
 {
   // Set dimensions of sum/sum_sq hyperslab to store
-  hsize_t count[] {n_filter, n_score, 2};
-  hid_t dspace = H5Screate_simple(3, count, nullptr);
+  constexpr int ndim = 3;
+  hsize_t count[ndim] {n_filter, n_score, 2};
```

Comment:
What was wrong with leaving the brackets empty?

Reply:
I did this just as a readability change. We needed the `ndim` variable anyway so I figured it could serve as a sort of hint of how to interpret the `count` and `start` variables. Let me know if you want me to revert that.